cmd/skipper/main.go

```diff
@@ -61,6 +61,8 @@ const (
 	sourcePollTimeoutUsage         = "polling timeout of the routing data sources, in milliseconds"
 	insecureUsage                  = "flag indicating to ignore the verification of the TLS certificates of the backend services"
 	proxyPreserveHostUsage         = "flag indicating to preserve the incoming request 'Host' header in the outgoing requests"
+	idleConnsPerHostUsage          = "maximum idle connections per backend host"
+	closeIdleConnsPeriodUsage      = "period of closing all idle connections in seconds. Not closing when less than 0"
```

Comment:
Seeing below that this is an actual Duration I was wondering if it wouldn't cause less surprise and doc hunting if it's actually named "Duration".
We could also accept a string as the parameter and user ParseDuration to give the end user great flexibility (We could still default to seconds for plain integers)